test/README.md

```diff
@@ -73,6 +73,31 @@ environment.
    So changes in the `test/drenv` package are available immediately
    without installing the package again.
 
+### Testing that `drenv` is healthy
+
+1. Change directory to the `test` directory:
+
+   ```
+   cd test
+   ```
+
```

Comment:
Serial numbers are to be updated on this and next point

Reply:
This is an ordered list - the rendered output will have the correct number, as you can see here:
https://github.com/RamenDR/ramen/blob/524031805d0943a45f63d325290526d58ddc65bf/test/README.md

It would be better if the text was using the right numbers, so reading the markdown
text will be more useful, but our markdown linter forbids this. There is also a good
reason to forbid it, it is easier to mainer when you don't have to update the numbers
manully when adding or removing items.

Reply:
oh! that make sense!